src/graph/graph.c

```
@@ -1408,6 +1408,105 @@
+	//--------------------------------------------------------------------------
+	// Graph's label matrices
+	//--------------------------------------------------------------------------
+
+	int n_lbl = Graph_LabelTypeCount(g);
+
+	for(LabelID lbl = 0; lbl < n_lbl; lbl++) {
+		D = Graph_GetLabelMatrix(g, lbl);
+		M = Delta_Matrix_M(D);
+
+		info = GxB_Matrix_memoryUsage(&n, M);
+		ASSERT(info == GrB_SUCCESS);
```

Cppcheck (2.10-2)

[error] Null pointer dereference

(nullPointer)